DeviceValidation/Validators/DeviceValidator.cs

```diff
@@ -7,7 +7,7 @@ namespace DeviceV2.Validators
 {
     public class DeviceValidator : AbstractValidator<Device>
     {
-        public DeviceValidator()
+        public DeviceValidator(string productID)
```

Comment:
Consumers of this validator are expecting to use IoC and inject an `IValidator<Device>` (which `AbstractValidator<Device>` implements). I believe your change of constructor here will make that difficult to do. What can we do to fix that?

Reply:
Ah, I see it one-level down. We are good.